autorel/clients/github_client.py

```diff
@@ -15,46 +15,46 @@
 
 class Github:
 
-    '''
+    """
         Client interacting with the Github REST API
-    '''
+    """
 
     def __init__(self, endpoint, port, token_or_login, password, clientID,
                  clientSecret, userAgent, timeout):
-        self._authorizationHeaders = {}
+        self._headers = {}  # HTTP headers
```

Comment:
No it is an instance attribute( `self` is used to access it) and this is because a generic client( here `HTTPClient` )  does not do any authorization stuff. The API specifically does require the authorization( here Github requires authorization and  the authorization headers are set depending on the API docs ). Apart from headers Github requires some extra headers to set, eg. `User-Agent`. Hence, we pass these extra headers to the `HTTPClient` instance as we are using aggregation principle of inheritance.